### PR TITLE
Added additional device properties

### PR DESCRIPTION
**Pull request by vauriga**
New properties: battery level and KeyPad battery critical

### CODE REVIEW

File: pynuki/device.py [hunk 1]
```diff
@@ -22,6 +22,14 @@ def nuki_id(self):
     def battery_critical(self):
         return self._json.get("batteryCritical")
 
+    @property
+    def battery_charge(self):
+        return self._json.get("batteryChargeState")
```
Comment (pschmitt) [inline pynuki/device.py:27]: I don't believe that this here is the right place for these attributes. It seems only the keypad has these. What I want to see is a dedicated class (inheriting from `pynuki.device`)

**Review by pschmitt — CHANGES_REQUESTED** [review]
Thanks for your contribution. See my comment in the review.

File: pynuki/device.py (hunk 1 above)
Comment (vauriga) [reply pynuki/device.py:27]: Hmm, but why? batteryChargeState is a battery level of the lock in percents, it shows after enabling inside Nuki App,
as for the Keypad... it's not a separate device, API returns it's battery critical state in the same json as the lock if it was paired with it. Both of those properties are part of the API response for the lock.

Comment (vauriga) [reply pynuki/device.py:27]: or we can move this to `lock.py`, like the `doorSensorStates`, what do you think?

Comment (pschmitt) [reply pynuki/device.py:27]: I was under the impression that the keypad was another `Lock` device, that just had a few keypad specific properties. If that's not the case as you suggest then moving it to `lock.py` makes more sense.

Comment (vauriga) [reply pynuki/device.py]: unfortunately no, it's just an extension to the lock and can't be used as standalone device.

Comment (steinerl) [reply pynuki/device.py]: IMO it would also be nice if the property names matched the key names in the JSON.

**Pull request merged**